src/primitives/decode.rs

```diff
@@ -180,7 +180,7 @@ impl<'s> UncheckedHrpstring<'s> {
     ///
     /// let mut unchecked = UncheckedHrpstring::new(&addr).unwrap();
     /// let witness_version = unchecked.remove_witness_version().unwrap();
-    /// assert_eq!(witness_version, Fe32::Q);
+    /// assert_eq!(witness_version, Fe32::ZERO);
```

Comment:
In f2df1282a553bd019e98adaffe9759eb900d0396:

Everywhere it is used as a witness version it should still be `Q` because that is what shows up in the actual encoded address.

Reply:
Done